tests/fixtures/.tests-config.json

```diff
@@ -1,4 +1,4 @@
 {
-    "check_legacy_language_path": true,
+    "true_config": true,
```

Comment:
what exactly is `true_config`, the name is not really clear to me. 
`config.is_enabled("true_config") is True`